Cargo.toml

```diff
@@ -121,6 +122,7 @@ lru = "0.12"
 memmap2 = "0.9"
 memory-stats = "1"
 multiaddr = "0.18"
+multihash = { version = "0.18", features = ["identity"] }
```

Comment:
`multihash` is re-exported from `cid` and can be accessed by `cid::multihash`, let's not add it to the deps

Reply:
fixed